Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1

```diff
@@ -25,8 +25,15 @@ Write-Verbose "breakBuild = $breakBuild"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
 
-. ./Common/SonarQubeHelpers/SonarQubeHelper.ps1
-. ./SonarQubePreBuildImpl.ps1
+. $PSScriptRoot/Common/SonarQubeHelpers/SonarQubeHelper.ps1
```

Comment:
I'm converting all script imports to use $PSScriptRoot as the tests will be able to import the product scripts as well.